test/toNumber.js

```diff
@@ -0,0 +1,62 @@
+import { assert } from 'chai';
+
+import BigInt from './shared/BigInt';
+import * as RA from '../src';
+
+describe('ToNumber', function () {
+  context('given a number', function () {
```

Comment:
If there are multiple `context` blocks in the same level can we please insert single empty line between them?

```js
context('desc1', function () {
});
context('desc2', function() { 
});
```
vs 

```js
context('desc1', function () {
});

context('desc2', function() { 
});
```